### DISCUSSION TITLE
How to use Stride in Avalonia UI

### DISCUSSION THREAD

**Discussion by martinrhan** [AvaloniaUI/Avalonia#15585] [Q&A]
https://gist.github.com/westonsoftware/a3fa982397fe1817ece4a27d3cbc5a89
Here is an example of that, however, as the author said, that copy data from Stride then send to Avalonia, it's not the best way. 
I want to know how to expose a texture in Avalonia to Stride and let Stride render on that texture.

**Comment by maxkatz6** [comment]
> that copy data from Stride then send to Avalonia, it's not the best way

Yes. In general, such things should be possible to implement with our GPU interop. That would allow embedding of 3D graphics into Avalonia app. You can check here https://github.com/AvaloniaUI/Avalonia/tree/master/samples/GpuInterop samples.
But it would require extensive knowledge of rendering stack of both Avalonia and Stride.
